### PR TITLE
Fixed release details and copyright year and pinned dependencies

### LINKED ISSUE #413: Outdated documentation
* Populus Version: 2.2.0
* OS: linux

### What was wrong?

The documentation on http://populus.readthedocs.io/en/v2.2.0/ is totally outdated and not useful anymore. The title says Populus 2.0.0-alpha.1 which is from on Feb 1.
I had to go to a lot of src code for:
 - deployment scripts
 - adding own chains like a parity chain

### PR DESCRIPTION
**Pull request by mandarvaze**
Fixes #413

### What was wrong?

The documentation still says "2.0.0-alpha.1"

### How was it fixed?

Changed the release to "2.2.0"
(Also changed the copyright year - although it is not related to #413)

#### Cute Animal Picture

jessica-weiller-93131-unsplash

(Photo by Jessica Weiller on Unsplash)

### CODE REVIEW

**Comment by miohtama** [comment]
Looks like PR fails, because someone has messed around with PyPi packages?

**Comment by veox** [comment]
Populus currently lacks a project maintainer. That's why the dates/dependencies are "out of sync", and why the Travis build fails.

File: docs/conf.py [hunk 1]
```diff
@@ -66,16 +66,16 @@ def __getattr__(cls, name):
...
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = "2.0"
+version = "2.2"
 # The full version, including alpha/beta/rc tags.
-release = "2.0.0-alpha.1"
+release = "2.2.0"
```
Comment (carver) [inline docs/conf.py]: The best option here would be to make the versions automatic. web3.py has an example of detecting the version:

https://github.com/ethereum/web3.py/blob/b058b26/docs/conf.py#L20-L27

and processing those into `version` and `release`: 

https://github.com/ethereum/web3.py/blob/b058b26/docs/conf.py#L65-L67

Are you interested in submitting that change?

**Comment by veox** [comment]
Might also want to add "pinned dependencies" to the PR title, so it's easier to find later. ;)

**Pull request merged**